readme.md

```diff
@@ -520,7 +520,13 @@ AVA, not Ava or ava. Pronounced [`/ˈeɪvə/` ay-və](media/pronunciation.m4a?ra
 
 ### Concurrency vs. parallelism
 
-Concurrency is not parallelism. It enables parallelism. It's about dealing with, while parallelism is about doing, lots of things at once.
+Concurrency is not parallelism. It enables parallelism.
+
+**Concurrency** is when two tasks can start, run, and complete in overlapping time periods. It doesn't necessarily mean they'll ever both be running at the same instant. (e.g. multitasking on a single-core machine.) 
```

Comment:
1. *two or more tasks*?
2. I wouldn't say *starting* and *completing* is necessary
3. I'm not a native speaker, but I wouldn't use the word `when` in that context to express a condition.